ballerina/http_log_manager.bal

```diff
@@ -35,9 +35,13 @@ public type TraceLogAdvancedConfiguration record {|
 # Represents HTTP access log configuration.
 #
 # + console - Boolean value to enable or disable console access logs
+# + format - The format of access logs to be printed (either `flat` or `json`)
+# + attributes - The list of attributes of access logs to be printed
 # + path - Optional file path to store access logs
 public type AccessLogConfiguration record {|
     boolean console = false;
+    string format = "flat";
```

Comment:
We only support flat and json formats right?
```suggestion
    "flat"|"json" format = "flat";
```

Reply:
Yeah, it should be this, but this gives the below error:
`error: configurable variable 'accessLogConfig.format' with type '("flat"|"json")' is not supported`.

Reply:
Shall we check with lang team regarding this? If this is a limitation according to the spec, we can go ahead with string type, else lets create a lang issue to improve the behaviour

Also, if this a string type, how are we handling the values other than `flat` or `json`. Do we give a warning?